.travis.yml

```diff
@@ -13,6 +13,7 @@ script:
 deploy:
   - provider: script
     script: ./deploy.sh
+    skip_cleanup: true
```

Comment:
Why?

Reply:
This is additional fix to https://travis-ci.org/aeternity/infrastructure/builds/311803848#L1087
We setup the callback configuration in [script phase](https://github.com/aeternity/infrastructure/blob/630f2f1b1af1000f19250a74f274752c9c9caedc/.travis.yml#L8) however travis does the git reset before deploy if we don't disable that with skip_cleanup.

Reply:
Thanks for explanation. I see I do not need to re-approve.